### ISSUE TITLE
ng2-handsontable very basic sample with minimum requirement

### ISSUE THREAD

**Issue by luking4** [valor-software/ng2-handsontable#104]
Hi,
Is it possible to create Demo with very simple example?
For instance,
1. One .ts file importing handsontable, providing selector for handsontable grid in it and same file containing minimum json data (2/3 columns).
2. Second .ts file importing first.ts file and bootstraping.
3. .html file using handsontable grid selector.
It would be really helpful, since current example is more complex and involves advance features. Basic demo is there but it is mixing with others on top (Index.html level).

**Comment by unoti** [comment]
Doing something really basic like what's happening here would be great: https://docs.handsontable.com/pro/1.6.0/tutorial-quick-start.html

I'm struggling to figure out how to get a minimal app working myself.  The trouble starts with the example that says:

`import {handsontable} from '../../../components/index';` 

I'm trying to piece together where to import handsontable from.  The data is generated in another place, and I'm general it's taking me a good amount of research to make a minimal version of this work.

**Comment by jwood803** [comment]
There's no `components/index` file anywhere in this repo so it's hard to figure out what's going on to reference as an example.

**Comment by crazypigs** [comment]
The instructions are very hard to follow. Fix the instructions man.

**Comment by renehamburger** [comment]
We've updated the package and the documentation completely. It works well with `angular@2.4.x` now.
